src/components/navbar/navbar.component.tsx

```diff
@@ -62,6 +65,43 @@ const Navbar: React.FC<NavbarProps> = ({
     };
   }, []);
 
+  let [searchBarIsVisible, setSearchBarIsVisible] = React.useState<any>(false);
+
+  let [closeSearchBar, setCloseSearchBar] = React.useState<any>(false);
+
+  const setSearchBar = () => {
```

Comment:
The whole search thing currently lives in em-home. Place all of this in an extension coming from there.

Reply:
@FlorianRappl  do you have an example of how i should put them in an extension ?